.github/workflows/docs-shim.yaml

```diff
@@ -0,0 +1,85 @@
+name: CI Docs
+
+on:
+  pull_request:
+    types: [milestoned, opened, reopened, synchronize]
+    paths:
+      - "**.md"
+      - "**.jpg"
+      - "**.png"
+      - "**.gif"
+      - "**.svg"
+      - docs/**
+      - .vscode/**
+      - .gitignore
+      - renovate.json
+      - .release-please-config.json
+      - release-please-config.json
+      - CODEOWNERS
+      - LICENSE
+
+jobs:
+  check-only-docs-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      only_docs_changed: ${{ steps.set_output.outputs.only_docs_changed }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Check for non-docs changes
+        id: docs_check
+        uses: ./.github/actions/check-non-docs-changes
+        with:
+          base_ref: ${{ github.base_ref }}
+
+      - name: Set outputs
+        id: set_output
+        run: echo "only_docs_changed=${{ steps.docs_check.outputs.non_docs_changed }}" >> $GITHUB_OUTPUT
+
+  lint-check:
+    needs: check-only-docs-changes
+    if: ${{ needs.check-only-docs-changes.outputs.only_docs_changed == 'false' }}
+    name: Lint Check(Simulated)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Lint Check
+        run: echo "Simulating lint-check job - Success!"
+
+  check-paths:
+    needs: check-only-docs-changes
+    if: ${{ needs.check-only-docs-changes.outputs.only_docs_changed == 'false' }}
+    name: Check Paths(Simulated)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Paths
+        run: echo "Simulating check-paths job - Success!"
+
+  run-package-test:
+    needs: check-only-docs-changes
+    if: ${{ needs.check-only-docs-changes.outputs.only_docs_changed == 'false' }}
+    name: Package Test(Simulated)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Package Test
+        run: echo "Simulating run-package-test job - Success!"
```

Comment:
Discussed offline, but I'm not sure this actually handles the original issue in the way we'd want. Our repo has required checks for `Schedule (all, <flavor>, <install/upgrade>) / Test`, so we'd need tasks named similarly here (with the matrix) in order to satisfy those required checks.

Also think we can ignore slim-dev and single-package for now since those aren't required and just always run lint-check (at some point that may encompass docs linting/spellcheck).